bpf/utils.h

```diff
@@ -253,12 +253,13 @@ static inline long pkt_drop_lookup_and_update_flow(struct sk_buff *skb, flow_id
                                                    enum skb_drop_reason reason) {
      flow_metrics *aggregate_flow = bpf_map_lookup_elem(&aggregated_flows, id);
      if (aggregate_flow != NULL) {
+         aggregate_flow->end_mono_time_ts = bpf_ktime_get_ns();
```

Comment:
Previously, there was some specific handling of empty start time here:
```
        // it might happen that start_mono_time hasn't been set due to
        // the way percpu hashmap deal with concurrent map entries
        if (aggregate_flow->start_mono_time_ts == 0) {
            aggregate_flow->start_mono_time_ts = current_time;
        }
```

cf also https://github.com/netobserv/netobserv-ebpf-agent/commit/c696bb0b7e4243845fe5289ac4a07727df517401

But I admit I don't really understand this old patch... I don't see in which cases `start_mono_time_ts` could not be set.

Reply:
@Joel this drop handling function

Reply:
but u are right that part is missing in flows.c the revert didn't bring it back